runtime/cudaq/platform/default/rest/helpers/iqm/iqm.config

```
@@ -0,0 +1,40 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# Tell DefaultQuantumPlatform what QPU subtype to use
+PLATFORM_QPU=remote_rest
+
+# Tell NVQ++ to generate glue code to set the target backend name
+GEN_TARGET_BACKEND=true
+
+# Add the rest-qpu library to the link list
+LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
+
+# Define the lowering pipeline, here we lower to Base QIR
+PLATFORM_LOWERING_CONFIG="expand-measurements,unrolling-pipeline,func.func(lower-to-cfg),canonicalize,func.func(quake-multicontrol-decomposition),iqm-gate-set-mapping"
+
+# Tell the rest-qpu that we are generating IQM JSON.
+CODEGEN_EMISSION=iqm
+
+# Physical backends must set library mode to false.
+LIBRARY_MODE=false
+
+PLATFORM_EXTRA_ARGS=""
+while [ $# -ne 0 ]; do
+	case "$1" in
+	--iqm-server-url)
+		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;url;$2"
+		shift
+		;;
+	--iqm-qpu-architecture)
```

Would you be willing to change the following to make the interfaces across the HW providers more consistent? Both IonQ and Quantinuum have similar options. This change would also necessitate other changes as part of the PR (not listed here), but I think it would be worthwhile to have consistent interfaces when possible.
```suggestion
	--iqm-machine)
```

---

Changed the CLI argument, keeping the internal `qpu-architecture` var.